.env.default

```diff
@@ -1,3 +1,8 @@
 ACCOUNT_ID=
 USERNAME=
 PASSWORD=
+
+WEBRTC_SIPX_NUMBER=+19192892727
+VOICE_NUMBER=
+VOICE_APP_ID=
+VOICE_CALLBACK_URL=
```

Comment:
I thought that @nguyer told me in a prior PR (my review of his work) that these env vars are no longer needed, which is why the readme was edited as well, but I could be wrong, I no longer have high familiarity with this code, but thought I would ask, to make sure we aren't undoing->redoing changes.